src/App.js

```diff
@@ -1,9 +1,90 @@
 import React from 'react';
 import './App.css';
+import Button from "./Button.js";
+import bag1 from "./assets/bag_1.png"
+import bag2 from "./assets/bag_2.png"
+import bag3 from "./assets/bag_3.png"
+import bag4 from "./assets/bag_4.png"
+import Product from "./Product.js";
+import brand from "./assets/brand.png";
+import ourStory from "./assets/our_story.png"
+import Tile from "./Tile.js";
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
+      <div>
+        <h1>Handbags & Purses</h1>
+        <nav>
+          <Button
+              typeOfButton="button"
+              buttonMessage="To the collection"
+              buttonTitle="To the collection"
+              enabled={false}
+          />
+          <Button
+              typeOfButton="button"
+              buttonMessage="Shop all bags"
+              buttonTitle="Shop all bags"
+              enabled={false}
+          />
+          <Button
+              typeOfButton="button"
+              buttonMessage="Pre-orders"
+              buttonTitle="Pre-orders"
+              enabled={true}
+          />
```

Comment:
Hierboven zeg je steeds: buttonMessage en buttonTitle maar deze zijn in alle gevallen hetzelfde. Opzich niet fout maar dubbel typwerk.